atomic_reactor/util.py

```diff
@@ -708,6 +725,16 @@ def get_manifest_digests(image, registry, insecure=False, dockercfg_path=None,
             insecure=insecure, dockercfg_path=dockercfg_path,
             version=version)
 
+        # If the registry has a v2 manifest that can't be converted into a v1
+        # manifest, the registry fails with status=400, and a error code of
+        # MANIFEST_INVALID.
+        if response.status_code == 400:
```

Comment:
Does this happen only when requesting version v1? If so, maybe also check if `version == 'v1'` ?